library/classes.lisp

```diff
@@ -113,9 +113,9 @@
   (repr :enum)
   (define-type Ord
     "The result of an ordered comparison."
-    LT
-    EQ
-    GT)
+    (LT "Less than")
```

Comment:
I'm happy to take this out but just wanted to leave it in for CI as proof of concept.